.circleci/config.yml

```diff
@@ -69,11 +69,30 @@ jobs:
       - run:
           name: Bundle Install
           command: bundle check || bundle install
-
       - run:
           name: Lint using rubocop
           command: bundle exec rubocop
-
+    - run:
+        name: Validate API specification
+        command: |
+          sudo npm install -g yaml-lint openapi-enforcer-cli
+          result=$(yamllint openapi.yml)
```

Comment:
Do we need this part?  openapi-enforcer errors if it's not valid yaml.

Reply:
No. Removed.